src/Uno.UI/UI/Xaml/Window/Native/NativeWindowWrapperBase.cs

```diff
@@ -181,20 +180,31 @@ public PointInt32 Position
 		}
 	}
 
-	public SizeInt32 Size
+	public SizeInt32 Size { get; private set; }
+
+	public SizeInt32 ClientSize { get; private set; }
+
+	protected void SetSizes(SizeInt32 size, SizeInt32 clientSize)
```

Comment:
I'm guessing this is a setup for future work to differentiate between clientSize and size? Because I didn't find any target where size and clientSize are not the same.